data-prepper-plugins/sns-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/sns/SnsSinkConfig.java

```diff
@@ -34,13 +33,12 @@ public class SNSSinkConfig {
     @Valid
     private AwsAuthenticationOptions awsAuthenticationOptions;
 
-    @JsonProperty("topic")
+    @JsonProperty("topic_arn")
     @NotNull
-    @NotEmpty
     private String topicArn;
 
     @JsonProperty("message_group_id")
-    private String messageGroupId;
+    public String messageGroupId;
```

Comment:
This should be `private` and provide a public getter - `getMessageGroupId()`.

Reply:
Thanks @dlvenable for your feedback, we have incorporated the changes and available for review.